topics/icrc_29_window_post_message_transport.md

```diff
@@ -36,12 +36,11 @@ using the [window.postMessage](https://developer.mozilla.org/en-US/docs/Web/API/
 ## Authentication
 
 * The relying party must authenticate the signer by the `origin` of the window that it opens.
-* The signer must authenticate the relying party by the `origin` property of the `message` event that it received.
```

Comment:
While the `ready` message is indeed a broadcast, it is still true that the signer authenticates the relying party by message event origin.
It just means that the ready message has to be broadcast _before_ the origin is known.
I would leave this sentence as is, given this is the "Authentication" section.

Reply:
Generally speaking yes but, this document specifies ICRC-29. In this particular case the signer cannot authenticate the origin. Therefore I personnaly would adapt those chapter "Authentication" and "Sending Messages" to the particular spec it relates but, of course, this PR is just a suggestion.